plugins/UM3NetworkPrinting/src/Cloud/Models/CloudClusterPrinterStatus.py

```diff
@@ -55,6 +55,7 @@ def __init__(self, enabled: bool, firmware_version: str, friendly_name: str, ip_
     #  \param controller - The controller of the model.
     def createOutputModel(self, controller: PrinterOutputController) -> PrinterOutputModel:
         model = PrinterOutputModel(controller, len(self.configuration), firmware_version = self.firmware_version)
+        model.updateBuildplate(self.build_plate.type)
```

Comment:
I would do this better in the `updateOutputModel()` method, where the output model is actually updated

Reply:
That's true. I've changed it.

Reply:
After 2-3 days of trying to track that bug down my eyes went blind and I didn't even notice the other stuff being set a few lines further down the file

Reply:
we like our classes and methods coherent :)

Reply:
Teamwork!